### ISSUE TITLE
tensorflow.keras saving weights?

### ISSUE THREAD

**Issue by oskar347567** [tensorflow/tensorflow#37209]
I have a model that is run on a time series train set trying to predict stock prices. I never save or load the model or the weights but every time i run it with the save hyperparameters it keeps decreasing in accuracy. I don't understand why, is tensorflow.keras saving the weights or something?

**Comment by Saduf2019** [comment]
@oskar347567 
please provide ius with simple stand alone code to replicate the issue faced by you on our local, along with the Tensorflow version on which issue is faced.
Also share the error logs for us to help you resolve the issue.

**Comment by Saduf2019** [comment]
@oskar347567 
please update on the above comment.
